src/site/markdown/server/manage-webadmin.md

```diff
@@ -1136,6 +1136,33 @@ Warning: Canceling this task should be considered unsafe as it will leave the cu
 Warning: While we have been trying to reduce the inconsistency window to a maximum (by keeping track of ongoing events),
 concurrent changes done during the reIndexing might be ignored.
 
+### Subscribing a user to all of its mailboxes
+ 
+```
+curl -XPOST http://ip:port/users/{usernameToBeUsed}/mailboxes?task=subscribeAll
+```
+
+Will schedule a task for subscribing a user to all of its mailboxes.
```

Comment:
What's the goal of this task? What happens if a user deliberately wants to not subscribe to some of her folders?

Reply:
> What's the goal of this task? 

Due to partial renames TB do mess the subscription up, keeping subscribed some mailboxes that no longer exists, and keeping unsubscribed mailboxes that exists.

This results in 'grey unusable' folders. Updating the subscription to the real mailbox works.

Most users are unaware of what a subscription  is. And managing subscriptions manually is a fastidious task (some of our users have 1000+ mailboxes). 

As such having a task to do it automatically as an admin allows to easily solve such problems when reported.

> What happens if a user deliberately wants to not subscribe to some of her folders?

The scope is to only run if it is relevant for the given users (they complains about subscription related bugs).

People able to manage their subscription themselves are not affected.

That's why I decided that the task should be user scoped, to allow such flexibility.

Reply:
Thanks, I'm ok with these explanations, but they are clearly missing in this documentation.